sites/kit.svelte.dev/src/routes/+layout.svelte

```diff
@@ -24,6 +24,9 @@
 				selected={$page.url.pathname.startsWith(`${base}/docs`) || undefined}
 				href="{base}/docs">Docs</NavItem
 			>
+			
+			<NavItem external="https://learn.svelte.dev/tutorial">Learn</NavItem>
```

Comment:
I like the idea of adding learn.svelte.dev to the navbar, but think the text "Learn" for the anchor might be a bit vague and perhaps we should call it "Tutorial". what do you think @Rich-Harris ?

Reply:
I got "Learn" from what the NextJS docs use. But yeah, "Tutorial" looks better for me. I'm committing the change until review for now.